SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -390,8 +390,8 @@ void USpatialPlayerSpawner::RetryForwardSpawnPlayerRequest(const Worker_EntityId
 		return;
 	}
 
-	Schema_CommandRequest* OldRequest = OutgoingForwardPlayerSpawnRequests.FindAndRemoveChecked(RequestId).Get();
-	Schema_Object* OldRequestPayload = Schema_GetCommandRequestObject(OldRequest);
+	const auto OldRequest = OutgoingForwardPlayerSpawnRequests.FindAndRemoveChecked(RequestId);
```

Comment:
I think we generally prefer non-autos outside loops.
How horrendous is the type for this one?

Reply:
`TUniquePtr<Schema_CommandRequest, ForwardSpawnRequestDeleter>`
Generally I wouldn't refer to things with custom deleters / allocators etc manually as it's mostly noise. Could also use a typedef In this case it would be a typedef for this one line so it seemed easier to use auto.

Could also fix it up to rely on SpatialView types, as it has a command request object that does this by default but I didn't want to refactor the whole thing or depend on spatial view for it.

Reply:
Happy to change it to something else if everyone else prefers.